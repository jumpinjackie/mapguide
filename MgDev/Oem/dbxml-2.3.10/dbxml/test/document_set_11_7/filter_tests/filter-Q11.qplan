<XQuery>
  <DocumentOrder>
    <DbXmlNav>
      <QueryPlanFunction result="collection" container="11.7.2.dbxml">
        <OQPlan>n(P(node-element-presence-none,=,c),P(node-element-presence-none,=,d),P(node-element-presence-none,=,a),P(node-element-presence-none,=,b))</OQPlan>
      </QueryPlanFunction>
      <DbXmlStep axis="descendant" name="a" nodeType="element"/>
      <DbXmlStep axis="child" name="b" nodeType="element"/>
      <DbXmlStep axis="child" name="c" nodeType="element"/>
      <DbXmlFilter>
        <DbXmlStep axis="child" name="d" nodeType="element"/>
      </DbXmlFilter>
    </DbXmlNav>
  </DocumentOrder>
</XQuery>
