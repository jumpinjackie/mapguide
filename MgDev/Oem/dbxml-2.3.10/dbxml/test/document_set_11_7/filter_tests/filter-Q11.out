<c><d>lala</d></c>